lib/matplotlib/table.py

```diff
@@ -361,6 +386,21 @@ def __getitem__(self, position):
 
     @property
     def edges(self):
+        """
+        The default value of `~.CustomCell.visible_edges` for newly added
+        cells using `.add_cell`.
+
+        Notes
+        -----
+        This setting does currently only affect newly created cells using
+        `.add_cell`.
+
+        To change existing cells, you have to set their edges explicitly:
```

Comment:
Use a code block instead?
```
blablah::

    the code
```
(if you really want to leave this in doctest format (ugh), note that the second line should have a ... prompt, not >>>)